staging/src/k8s.io/kubectl/pkg/util/i18n/i18n.go

```diff
@@ -52,6 +52,11 @@ var knownTranslations = map[string][]string{
 	},
 }
 
+func init() {
+	// LoadTranslations must occur in init so that package-level variables initialized with strings can be translated
+	LoadTranslations("kubectl", nil)
```

Comment:
A possible issue that occurs to me with this approach is library usage. In theory, a library consumer constructing their own commands might want to supply their own `getLanguageFn` (`func LoadTranslations(root string, getLanguageFn func() string) error `), or not load translations at all.

Perhaps an alternative could be to have `i18n.T` itself lazy-load the translations? If there isn't an easy way to detect whether they're already loaded, `LoadTranslations` itself, or a wrapper charged with managing this, could set one?

Reply:
I added lazy loading.

I exposed it through a `SetLoadTranslationsFunc()` function that can be used by libraries if they want to substitute their own lazy loading function.  They could still call `LoadTranslations` too.

Its a little more complicated than I was going for initially, but I think it should work.